alpha/alpha-server/src/main/resources/application.yaml

```diff
@@ -146,6 +146,46 @@ spring:
       eclipselink:
         ddl-generation: none
 
+---
+spring:
+  profiles: rabbit
+  cloud:
+    stream:
+      binders:
+        defaultRabbit:
+          type: rabbit
+          environment:
+            spring:
+              rabbitmq:
+                host: 127.0.0.1
```

Comment:
It's better to provide the default setting as docker compose does.

Reply:
Please remove the rabbit default configuration in the application.yml file and add rabbit configuration description to the document https://github.com/apache/servicecomb-pack/blob/master/docs/fsm/eventchannel_zh.md

Reply:
done